static/templates/user_profile_modal.hbs

```diff
@@ -29,7 +32,15 @@
                                     </div>
                                     <div id="user-type" class="default-field">
                                         <div class="name">{{#tr}}Role{{/tr}}</div>
-                                        <div class="value">{{user_type}}</div>
+                                        {{#if is_bot}}
+                                            {{#if is_system_bot}}
+                                            <div class="value">{{#tr}}System bot{{/tr}}</div>
+                                            {{else}}
+                                            <div class="value">{{#tr}}Bot{{/tr}}</div>
+                                            {{/if}}
+                                        {{else}}
+                                            <div class="value">{{user_type}}</div>
+                                        {{/if}}
```

Comment:
This block looks like it duplicates one that already exists in `user_info_popover_content.hbs`; probably should be deduplicated. It's fine to do this in a follow-up PR, since it'll require some care to do.